lib/plugins/object.js

```diff
@@ -0,0 +1,138 @@
+const fp = require('fastify-plugin');
+
+const OBJECTS_TABLE = 'warehouse-objects';
+const OBJECT_VARIANTS_TABLE = 'warehouse-object-variants';
+const OBJECTS_HISTORY_TABLE = 'warehouse-objects-history';
+
+module.exports = fp(
+  async function (fastify) {
+    const { dynamo } = fastify;
+
+    fastify.decorate('getObject', async function ({ name, env }) {
+      const params = {
+        Key: { name, env },
+        TableName: OBJECTS_TABLE
+      };
+      const { Item: item } = await dynamo.get(params).promise();
+      return item;
+    });
+
+    fastify.decorate('putObject', function ({
```

Comment:
Should this be an `async function`?

Reply:
We do not `await` anything here. We return the promise directly, so no.